CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ### Internal
 * Using Core 10.3.0.
+* Migrated to Bison parser
```

Comment:
This should also have a user-facing note. Perhaps sync with @jedelbo on what would be an appropriate user-facing note about it. As far as I can see, the package is now 6% smaller, so perhaps something like:

> Replaced the implementation of the string query parser (the one used for [realm.All<Foo>().Filter("some-string-query")](https://docs.mongodb.com/realm-sdks/dotnet/10.0.0-beta.3/reference/Realms.CollectionExtensions.html#Realms_CollectionExtensions_Filter__1_System_Linq_IQueryable___0__System_String_)). This results in ~5% reduction of the size of the native binary while keeping the query execution times on par with the old parser. (**if Jørgen has other comments here, be sure to include them**).

Reply:
This is just fine